flake.nix

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
did you mean to checkin these flake* files?

Reply:
Aaand nope. I keep adding them accidentally.

Reply:
Fixed.

Reply:
(For context: I use those two files to easily start a development environment on macOS)